po/ru.po

```diff
@@ -1065,40 +1075,59 @@ msgid ""
 "  mutexes) and async/await concurrency (cooperative multitasking using\n"
 "  futures)."
 msgstr ""
+"* [Android](android.md): рассчитанный на полдня курс по использованию Rust для разработки на\n"
+"платформе Android (AOSP).\n"
+"Сюда входит взаимодействие с C, C++ и Java.\n"
+"* [Bare-metal](bare-metal.md): однодневное занятие по использованию Rust для bare-metal\n"
+"  (embedded)разработки. Охватываются как микроконтроллеры, так и прикладные процессоры.\n"
+"* [Concurrency](concurrency.md): полный день занятий по изучению конкуретности в Rust.\n"
+"Мы рассмотрим как классическую конкурентность\n"
+"(вытесняющее планирование с использованием потоков и мьютексов),\n"
+"так и async/await конкурентность (кооперативная многозадачность с использованием футур)."
 
 #: src/welcome.md:32
 msgid "## Non-Goals"
-msgstr ""
+msgstr "## За рамками курса"
 
 #: src/welcome.md:34
 msgid ""
 "Rust is a large language and we won't be able to cover all of it in a few days.\n"
 "Some non-goals of this course are:"
 msgstr ""
+"Rust - это объемный язык, и мы не сможем охватить его весь за несколько дней.\n"
+"Вещи, выходящие за рамки курса:"
 
 #: src/welcome.md:37
 msgid ""
 "* Learn how to develop macros, please see [Chapter 19.5 in the Rust\n"
 "  Book](https://doc.rust-lang.org/book/ch19-06-macros.html) and [Rust by\n"
 "  Example](https://doc.rust-lang.org/rust-by-example/macros.html) instead."
 msgstr ""
+"* Разработка макросов, см. [Chapter 19.5 in the Rust\n"
+"  Book](https://doc.rust-lang.org/book/ch19-06-macros.html) и [Rust by\n"
+"  Example](https://doc.rust-lang.org/rust-by-example/macros.html)."
 
 #: src/welcome.md:41
 msgid "## Assumptions"
-msgstr ""
+msgstr "## Предпосылки"
 
 #: src/welcome.md:43
 msgid ""
 "The course assumes that you already know how to program. Rust is a statically\n"
 "typed language and we will sometimes make comparisons with C and C++ to better\n"
 "explain or contrast the Rust approach."
 msgstr ""
+"Курс предполагает, что вы уже умеете программировать. Rust это\n"
```

Comment:
Done!

Reply:
Just a small detail in case it makes your life easier: `mdbook` will [turn `--` into an n-dash and `---` into an m-dash](https://rust-lang.github.io/mdBook/format/markdown.html?highlight=dash#smart-punctuation). I would probably lean towards taking advantage of this feature, but feel free to use "real" dashes instead if you like that better.